README.md

```diff
@@ -84,12 +84,44 @@ Few hints, to customize the [`docker-compose.yml`](docker-compose.yml) orchestra
 - [x] Given the about restart on failure is recommended as policy, this ensure that the scripts restart when it exit -1 and when the node where it is running is removed / drained (you need more than one master node!)
 - [x] The repository includes a basic set of Travis CI tests that tests the script behavior against basic conditions in a single swarm node and without data persistence (initialization, scale up, scale down)
 
+## Testing
+
+### Prerequisites
+
+* A [Docker Swarm cluster](https://docs.docker.com/engine/swarm/swarm-tutorial/create-swarm/) (docker >= 17.06.0-ce) (locally or in the cloud as you prefer).
+* [shunit2](https://github.com/kward/shunit2) installed on your client
+* (Optionally) [VirtualBox](http://virtualbox.org) if you want to use the scripts provided in `utils` folder.
+* Your Docker client configured to point to the Docker Swarm cluster.
+
+### Utilities
+To test the script you need to set-up a Docker Swarm cluster. Assuming that VirtualBox is installed on your Linux/MacOS system you, can use the scripts in the `utils` folder to create and manage the cluster:
+* [create-cluster.sh](utils/create-cluster.sh) creates a Docker Swarm cluster with 1 Manager and 2 Workers (you can change the number of the Workers)
```

Comment:
I believe this repo is not the right place for these scripts.
For the recipes, I've been using https://github.com/aelsabbahy/miniswarm, which I have referenced in the "utils" of the recipes repo.

Reply:
for the time being i would keep as it is. we have other priorities and this is enough for our needs. by using a different script, we will need either to change "fixed" names in the tests or to work out generic ways to retrieve the names of the docker machines to connect to. (which may not be easy if you have multiple docker-machine environments).